code-scanning/bandit.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
🚧 This will conflict with the changes of PR #2497 which includes this same change. If this change is accepted on this PR, please coordinate with PR #2497 (author willing to rebase if this is accepted first)

Reply:
Often such changes won't trigger conflicts as long as the changes are identical.

Reply:
I'll go ahead and mark this resolved. 🔰 Sorry for the confusion.

# TL;DR:
---

🙊 unfortunately [my changes move this line while changing it](https://github.com/actions/starter-workflows/pull/2497/files#diff-8c79107c6af62e0cea73b4518ed47f7bdb92b08f69ea9aa549854e5eaf0c6e21L32-R34) so I was worried about it.

🩹 Luckily when I tested it locally there was no issue here (when merging yours before mine, only an unrelated rebase was needed)

Script used to test and confirm:
```bash
cd /tmp
git clone https://github.com/actions/starter-workflows.git
cd ./starter-workflows
# add your fork from PR 2465
git remote add -t bump-actions -m main jsoref https://github.com/jsoref/starter-workflows.git
# add my fork from PR 2497
git remote add -t patch-1 -m main reactive-firewall https://github.com/reactive-firewall/starter-workflows.git
# fetch local copies of the forks
git fetch --all
checkout bump-actions
checkout patch-1
checkout main
# rebase fork from PR 2465 (and resolve with mains copy)
git rebase main bump-actions
git checkout main -- deployments/google-cloudrun-docker.yml  # this is not modified in my PR so exact resolution does not matter 
git checkout main -- deployments/google-cloudrun-source.yml  # this is not modified in my PR so exact resolution does not matter 
git rebase --continue
# make sure we are on main (the target branch of both PRs)
git checkout main
git merge bump-actions main  # works now with rebase
git merge patch-1 main  # no conflict in code-scanning/bandit.yml :tada:

```

Reply:
No worries and thanks for testing. I know that _sometimes_ it works but didn't have the energy to do the testing myself.

It mostly doesn't matter, I'm very comfortable rebasing such changes and wouldn't mind doing so if someone from github said that they were ready to actually look at the PR and that it had conflicts.